include/deal.II/lac/linear_operator.h

```diff
@@ -49,12 +49,7 @@ LinearOperator<Range, Domain> linear_operator (const Matrix &);
 template <typename Domain = Vector<double>,
           typename Range = Domain>
 LinearOperator<Domain, Range>
-null_operator(const std::function<void(Domain &, bool)> &,
-              const std::function<void(Range &,  bool)> &);
-
-template <typename Range = Vector<double>>
-LinearOperator<Range, Range>
-null_operator(const std::function<void(Range &, bool)> &);
+null_operator(const LinearOperator<Domain, Range> &);
```

Comment:
Out of curiosity, does it make sense to consider a null operator that has a different range than domain type?